tasks/fix-cat3.yml

```
@@ -32,6 +32,25 @@
       - complexity-high
       - passwd
 
+- name: |
+      "LOW | RHEL-07-021022 | PATCH | The Red Hat Enterprise Linux operating system must mount /dev/shm with the nodev option."
+      "LOW | RHEL-07-021023 | PATCH | The Red Hat Enterprise Linux operating system must mount /dev/shm with the nosuid option."
+      "LOW | RHEL-07-021024 | PATCH | The Red Hat Enterprise Linux operating system must mount /dev/shm with the noexec option."
+  mount:
+      path: /dev/shm
+      state: mounted
+      src: "{{ dev_shm_mount.device|default('tmpfs') }}"
+      fstype: "{{ dev_shm_mount.fstype|default('tmpfs') }}"
+      opts: "{{ dev_shm_mount_opts }}"
+  when: (rhel_07_021022 or rhel_07_021023 or rhel_07_021024)
+  vars:
+      dev_shm_mount: "{{ ansible_mounts | json_query('[?mount == `/dev/shm`] | [0]') }}"
+      dev_shm_mount_opts: "{{ dev_shm_mount.options|default('defaults') }},nodev,nosuid{{ (rhel_07_021024) | ternary(',noexec', '') }}"
```

Style consistency for j2 
```suggestion
      dev_shm_mount_opts: "{{ dev_shm_mount.options | default('defaults') }},nodev,nosuid{{ (rhel_07_021024) | ternary(',noexec', '') }}"
```